src/open_inwoner/haalcentraal/api_models.py

```diff
@@ -0,0 +1,37 @@
+import dataclasses
+from datetime import date
+from typing import Optional
+
+from open_inwoner.accounts.models import User
+
+
+@dataclasses.dataclass
+class BRPData:
+    # match with user model fields
+    first_name: str = ""
+    infix: str = ""
+    last_name: str = ""
+    street: str = ""
+    housenumber: str = ""
+    city: str = ""
+    birthday: Optional[date] = None
+
+    # extra fields
+    # TODO rename birthday_city
+    initials: str = ""
+    birthday_city: str = ""
+    gender: str = ""
+    # TODO rename postcode to postal_code
+    postcode: str = ""
+    # TODO rename land to country
+    country: str = ""
+
```

Comment:
- Remove comments
- `birthday_city` should really be `birth_place` or `place_of_birth`. See above.